### PR TITLE
CI: use fixed platform version instead of dev branch

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Review by Alizter — APPROVED** [review]

**Comment by Alizter** [comment]
Will merge when CI passes. Looks like something went wrong.

**Comment by Zimmi48** [comment]
The CI won't start. Looks like GitHub is having problems with both Actions and Webhooks right now: https://www.githubstatus.com/

File: .github/workflows/ci-windows.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
 
       - name: Download Platform
         env:
-          PLATFORM: "https://github.com/coq/platform/archive/main.zip"
+          PLATFORM: "https://github.com/coq/platform/archive/refs/tags/2022.01.0.zip"
```
[the remark this answers is not in the document]
Comment (MSoegtropIMC) [reply .github/workflows/ci-windows.yml]: What is wrong with the tag link? It works for me, it is what github Web UI says it is and I would prefer a tag over the commit to note that it is a tag.

[the remark this answers is not in the document]
Comment (Zimmi48) [reply .github/workflows/ci-windows.yml]: The problem with the tag is that https://github.com/coq/platform/pull/223 was not in the last release.

[the remark this answers is not in the document]
Comment (MSoegtropIMC) [reply .github/workflows/ci-windows.yml]: Yes, sorry, i just saw that. Indeed `88d69fcd97b313de1c6e39b9439234ad6e863f21` is what I would recommend for CI until 2022.03 is out.

**Review by MSoegtropIMC — APPROVED** [review]

**Comment by SkySkimmer** [comment]
green

**Review by Zimmi48 — APPROVED** [review]
@coqbot merge now (taking over assignment since this is urgent)

**Pull request merged**